frontend/src/components/common/AddAdminModal.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { useState } from "react";
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  Modal,
+  ModalHeader,
+  ModalCloseButton,
+  ModalContent,
+  ModalBody,
+  ModalFooter,
+  ModalOverlay,
+  Radio,
+  RadioGroup,
+  HStack,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { PlusOutlineIcon } from "./icons";
+
+const AddAdminFormLabel = ({ text }: { text: string }): React.ReactElement => {
+  return (
+    <FormLabel
+      as="legend"
+      color="blue.300"
+      mb={0}
+      fontSize="20px"
+      lineHeight="26px"
+    >
+      {text}
+    </FormLabel>
+  );
+};
+
+const AddAdminTextInput = ({
+  validationType,
+  placeholder,
+  handleChange,
+}: {
+  validationType: string;
+  placeholder: string;
+  handleChange: (val: string) => void;
+}): React.ReactElement => {
+  return (
+    <Input
+      type={validationType}
+      placeholder={placeholder}
+      onChange={(e) => handleChange(e.target.value)}
+      fontSize="18px"
+      width="320px"
+      height="48px"
+      textAlign="center"
+      backgroundColor="grey.100"
+      borderColor="grey.100"
+    />
+  );
+};
+
+const AddAdminModal = (): React.ReactElement => {
+  const { onOpen, onClose, isOpen } = useDisclosure();
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [confirmEmail, setConfirmEmail] = useState("");
+  const [hasJumpMathEmail, setHasJumpMathEmail] = useState<boolean | null>(
+    null,
+  );
+
+  const onModalClose = () => {
+    setHasJumpMathEmail(null);
+    onClose();
+  };
+
+  const onAddAdminClick = () => {
+    if (email !== confirmEmail) {
+      console.log("emails don't match");
```

Comment:
Are we supposed to show an error state here? If so, can we make sure that there is another ticket to add this functionality!